src/components/seo.js

```diff
@@ -28,6 +28,11 @@ const SEO = ({
       social: { twitter },
     },
   } = site;
+  const metaKeywords = keywords || [
```

Comment:
keywords would be an empty array by default which isn't a falsy value. Can we replace this to check for length instead?

Reply:
yep but only if default keywords props are not defined.